systemtest/src/test/java/io/strimzi/systemtest/AbstractST.java

```diff
@@ -265,46 +276,46 @@ protected void teardownEnvForOperator() {
      * @param namespace namespace where CO will be deployed to
      * @param bindingsNamespaces list of namespaces where Bindings should be deployed to
      */
-    public static void applyBindings(String namespace, List<String> bindingsNamespaces) {
+    public static void applyBindings(ExtensionContext extensionContext, String namespace, List<String> bindingsNamespaces) {
         for (String bindingsNamespace : bindingsNamespaces) {
-            applyClusterRoleBindings(namespace);
-            applyRoleBindings(namespace, bindingsNamespace);
+            applyClusterRoleBindings(extensionContext, namespace);
+            applyRoleBindings(extensionContext, namespace, bindingsNamespace);
         }
     }
 
     /**
      * Method for apply Strimzi cluster operator specific Role and ClusterRole bindings for specific namespaces.
      * @param namespace namespace where CO will be deployed to
      */
-    public static void applyBindings(String namespace) {
-        applyBindings(namespace, Collections.singletonList(namespace));
+    public static void applyBindings(ExtensionContext extensionContext, String namespace) {
+        applyBindings(extensionContext, namespace, Collections.singletonList(namespace));
     }
 
     /**
      * Method for apply Strimzi cluster operator specific Role and ClusterRole bindings for specific namespaces.
      * @param namespace namespace where CO will be deployed to
      * @param bindingsNamespaces array of namespaces where Bindings should be deployed to
      */
-    public static void applyBindings(String namespace, String... bindingsNamespaces) {
-        applyBindings(namespace, Arrays.asList(bindingsNamespaces));
+    public static void applyBindings(ExtensionContext extensionContext, String namespace, String... bindingsNamespaces) {
+        applyBindings(extensionContext, namespace, Arrays.asList(bindingsNamespaces));
     }
 
-    private static void applyClusterRoleBindings(String namespace) {
+    private static void applyClusterRoleBindings(ExtensionContext extensionContext, String namespace) {
         // 021-ClusterRoleBinding
-        KubernetesResource.clusterRoleBinding(TestUtils.USER_PATH + "/../packaging/install/cluster-operator/021-ClusterRoleBinding-strimzi-cluster-operator.yaml", namespace);
+        ClusterRoleBindingResource.clusterRoleBinding(extensionContext, TestUtils.USER_PATH + "/../packaging/install/cluster-operator/021-ClusterRoleBinding-strimzi-cluster-operator.yaml", namespace);
```

Comment:
Should you use in these places something similar to `PATH_TO_PACKAGING_EXAMPLES` from `Constants.java`? You changed it there, maybe it can change here as well.

Reply:
Yes, I have added it thanks.